setup.cfg

```diff
@@ -30,22 +30,25 @@ test = pytest
 
 [tool:pytest]
 collect_ignore = ['setup.py']
-addopts = 
-	--cov
+norecursedirs = docs/notebooks/*
```

Comment:
```suggestion
```

Reply:
Explanation: setting `norecursedirs` will remove all other default settings. This was causing pytest to try to run the testing ensembles of packages installed alongside xscen in the tox directory. Lesson learned.